src/Python/sirf/contrib/NEMA/generate_nema_rois.py

```diff
@@ -0,0 +1,257 @@
+'''Generate NEMA ROIs.
+
+Usage:
+  generate_nema_rois [--help | options]
+
+Options:
+  -s <file>, --sino=<file>     raw data file [default: no default you need an input of a NEMA sinogram]
+  -o <out_path>, --outpath=<out_path>     path to data files, defaults to current directory
+                               subfolder of SIRF root folder
+  -xy <xy_size>, --xysize=<xy_size> optional size of image in x and y, default is 150
```

Comment:
```suggestion
  -xy <xy_size>, --xysize=<xy_size> optional size of image in x and y  [default: 150]
```
That way, `docopt` handles it.